hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -333,6 +334,7 @@ func MachineTemplateSpec(nodePool *hyperv1.NodePool, bootImage BootImage, hclust
 		Template: capikubevirt.KubevirtMachineTemplateResource{
 			Spec: capikubevirt.KubevirtMachineSpec{
 				VirtualMachineTemplate: *vmTemplate,
+				BootstrapCheckSpec:     capikubevirt.VirtualMachineBootstrapCheckSpec{CheckStrategy: "ssh"},
```

Comment:
@stevekuznetsov Would you mind setting the CheckStrategy to `none` instead of `ssh`. While `ssh` probably passes our presubmit right now, it is likely this will break for us in the future. We should have already been explicitly setting this to `none`. Sorry you encountered this.

Reply:
@davidvossel done - PTAL the last commit!